HW7(Tests)/unit.test.js

```diff
@@ -0,0 +1,70 @@
+const { checkPassword, checkEmail } = require('./HW7_Trupkina');
+const { describe, test, expect } = require('@jest/globals');
+
+describe('Testing... ', () => {
+    describe('Testing function checkPassword: ', () => {
+        let passwordTrue = [
+            'aaaaaaaaA#',
+            'AAAAAAA22a#',
+            'passwOrd3%'
+        ];
+        let passwordFalse = ['парОль3%',
+            'AAAAAAAAAA#',
+            'aA#',
+            '2222222222A#',
+            '2222222222a#',
+            '22222222222',
+            '2222222222#',
+            '############'
+        ];
```

Comment:
passwordTrue / passwordFalse - incorrect names.
better option will be smth like correctPasswords / incorrectPasswords